pkg/ssh/ssh.go

```diff
@@ -383,6 +383,7 @@ func (l *SSHTunnelList) removeAndReAdd(e sshTunnelEntry) {
 			l.entries = append(l.entries[:i], l.entries[i+1:]...)
 			l.adding[e.Address] = true
 			go l.createAndAddTunnel(e.Address)
+			go e.Tunnel.Close()
```

Comment:
This looks functionally correct, but why spawning a new goroutine for this task? Seems like you could modify this function to do the same without creating another goroutine.